src/app/src/mainwindow.cpp

```diff
@@ -815,3 +833,17 @@ void MainWindow::acquisitionThread(MainWindow* cls) {
     spdlog::info("Acquisition done, sending signal");
     emit cls->sig_acquisition_done();
 }
+
+
+
+/**
+ * Helper function that will take the line time for each mode from the config file and
+ * calculate the max frame rate based on the caputure reagion height.
+*/
+double MainWindow::calcMaxFrameRate(int p1,int p2,int16_t spdtable_index,double line_times [4],uint16_t spdtable){
```

Comment:
What's the difference between `spdtable_index` and `spdtable`? It doesn't look like `spdtable_index` is used anywhere.

Also when you pass arrays to functions you typically want to do it as a reference to the array instead of by value, so that the code doesn't have to make a copy of the whole array but in this case I think it would be better to pass in the single line time instead of the whole array and index so something like

```cpp
double MainWindow::calcMaxFrameRate(int p1, int p2, double line_time);
```

would probably make more sense